backend/TreeOfAKind.Domain/TreeOfAKind.Domain.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="MediatR" Version="9.0.0" />
+    <PackageReference Include="NodaTime" Version="3.0.1" />
```

Comment:
What's that

Reply:
Also it would be better to have single `csproj` where all versions would be held, and use only necessary in the Domain, Infrastructure etc.